record_web/lib/record_web.dart

```diff
@@ -145,15 +159,34 @@ class RecordPluginWeb extends RecordPlatform {
     return devices;
   }
 
+  double getMaxAmplitude() {
+    assert(analyser != null);
```

Comment:
check analyser instance and fallback to minAmplitude instead

Reply:
done